Website/DesktopModules/Hotcakes/Core/Admin/Reports/PaymentFailures/View.aspx

```diff
@@ -0,0 +1,131 @@
+﻿<%@ Page Language="C#" MasterPageFile="../../AdminNav.master" AutoEventWireup="true" CodeBehind="View.aspx.cs" Inherits="Hotcakes.Modules.Core.Admin.Reports.PaymentFailures.View" %>
+
+<%@ Register Src="../../Controls/MessageBox.ascx" TagName="MessageBox" TagPrefix="hcc" %>
+<%@ Register Src="../../Controls/DateRangePicker.ascx" TagName="DateRangePicker" TagPrefix="hcc" %>
+<%@ Register Src="../../Controls/Pager.ascx" TagName="Pager" TagPrefix="hcc" %>
+<%@ Register Src="../../Controls/NavMenu.ascx" TagName="NavMenu" TagPrefix="hcc" %>
+<%@ Register Src="../../Controls/ContactAbandonedCartUsers.ascx" TagName="ContactAbandonedCartUsers" TagPrefix="hcc" %>
+
+<asp:Content ContentPlaceHolderID="NavContent" runat="server">
+
+    <hcc:NavMenu runat="server" BaseUrl="reports/" ID="NavMenu" />
+
+    <div runat="server" id="divNavBottom">
+        <div class="hcBlock">
+            <div class="hcForm">
+                <div class="hcFormItem">
+                    <asp:LinkButton ID="lnkExportToExcel" CssClass="hcButton" resourcekey="ExportToExcel" runat="server" />
+                </div>
+                <div class="hcFormItem">
+                    <asp:LinkButton ID="lnkDownloadContacts" CssClass="hcButton" resourcekey="DownloadContacts" runat="server" />
+                </div>
+            </div>
+        </div>
+    </div>
+
+</asp:Content>
+
+<asp:Content ContentPlaceHolderID="MainContent" runat="Server">
+    <div class="hcReport">
+        <h1><%=PageTitle %></h1>
+        <hcc:MessageBox ID="ucMessageBox" runat="server" />
+
+        <div class="hcColumnLeft" style="width: 60%">
+            <div class="hcForm">
+                <hcc:DateRangePicker ID="ucDateRangePicker" runat="server" RangeType="ThisMonth" />
+            </div>
+        </div>
+
+        <div class="hcColumnRight hcLeftBorder" style="width: 39%">
+            <div class="hcForm">
+                <div class="hcFormItem">
+                    <asp:RadioButtonList ID="rblAgregatedReport" AutoPostBack="true" runat="server">
+                        <asp:ListItem Value="False" resourcekey="CartsReport" Selected="True" />
+                        <asp:ListItem Value="True" resourcekey="ProductsReport" />
+                    </asp:RadioButtonList>
+                </div>
+            </div>
+        </div>
+        <div class="hcForm">
+            <div class="hcFormItem">
+                <asp:Label ID="lblNoCartsMessage" resourcekey="NoPaymentFailuresFound" CssClass="hcInfoLabelLeft" Visible="false" runat="server" />
+            </div>
+        </div>
+
+        <asp:Panel ID="pnlReportsData" runat="server">
+            <asp:Repeater ID="rpCarts" runat="server">
+                <HeaderTemplate>
+                    <table class="hcGrid">
+                        <tr class="hcGridHeader">
+                            <th><%=Localization.GetString("OrderNumber") %></th>
```

Comment:
@alejoroman0605 Please make the order number link to the respective order.

Reply:
@WillStrohl I already added the link to the order number
![image](https://github.com/user-attachments/assets/c76bd654-c7e9-4d7e-99df-488600f26309)